images/operator/Dockerfile

```diff
@@ -2,6 +2,8 @@
 # SPDX-License-Identifier: Apache-2.0
 
 ARG BASE_IMAGE=scratch
+ARG CILIUM_BUILDER_IMAGE=quay.io/cilium/cilium-builder:f55691636922518ca40802f59d57f36ee4741c0d@sha256:4c21a3e21e9e07f0301191b3ae6921c8208a3910ce1b73be6ad445e544590e3a
```

Comment:
Can we move this ARG to line 74? Just to make it more obvious that this image is not used in production.

Reply:
good point -> moved the line

Reply:
It's unfortunate.